ramls/license.json

```diff
@@ -10,10 +10,17 @@
     "tags":{
       "type": "string"
     },
+    "name":{
+      "type": "string"
+    },
+    "description":{
+      "type": "string"
+    },
     "customProperties":{
       "type": "string"
     }
   },
   "required": [
+    "name"
   ]
```

Comment:
@ianibo @sosguthorpe If there are no required properties, then just omit the optional "required" array.